bin/proving-service/src/commands/worker.rs

```diff
@@ -10,10 +10,10 @@ use crate::{api::RpcListener, generated::api_server::ApiServer, utils::MIDEN_PRO
 #[derive(Debug, Parser)]
 pub struct StartWorker {
     /// The host of the worker
-    #[clap(short, long, default_value = "0.0.0.0")]
+    #[clap(short, long, default_value = "0.0.0.0", env = "WORKER_HOST")]
     host: String,
     /// The port of the worker
-    #[clap(short, long, default_value = "50051")]
+    #[clap(short, long, default_value = "50051", env = "WORKER_PORT")]
     port: u16,
 }
```

Comment:
I don't think these need to be read from the environment variables (i.e., they should remain command-line only).